js/api.ts

```diff
@@ -154,11 +200,14 @@ function fakeUserType(): "teacher" | "learner" {
 export function fetchPortalDataAndAuthFirestore(): Promise<IPortalRawData> {
   const offeringPromise = fetchOfferingData();
   const classPromise = fetchClassData();
-  return classPromise.then((classData: any) => {
-    const firestoreJWTPromise = fetchFirestoreJWT(classData.class_hash);
-    return Promise.all([offeringPromise, firestoreJWTPromise]).then((result: any) => {
-      const offeringData = result[0];
-      const rawFirestoreJWT = result[1].token;
+  return Promise.all([offeringPromise, classPromise]).then(([offeringData, classData]: [any, any]) => {
+    const resourceLinkId = offeringData.id.toString();
+    // only pass resourceLinkId if there is a studentId
+    // FIXME: if this is a teacher viewing the report of a student there will be a studentId
+    // but the token will be for a teacher, so then the resourceLinkId should be null
```

Comment:
Normally I'd say this shouldn't be merged with a `FIXME` but in this case the issue is with another feature which isn't part of this PR: supporting launching of an AP report from the portal.
So it seems fine to leave this in and we'll need to deal with it when implementing that feature.